js/main.js

```diff
@@ -77,17 +211,43 @@ function breedSearch(name) {
   for (var i = 0; i < name.length; i++) {
     breed += name.charAt(i).toLowerCase();
   }
-  var xhr = new XMLHttpRequest();
-  xhr.open('GET', 'https://dog.ceo/api/breed/' + breed + '/images/random');
-  xhr.responseType = 'json';
-  xhr.addEventListener('load', function () {
 
-    var newPic = renderPic(xhr.response);
-    $pictureList.appendChild(newPic);
+  if (picViewNumber === 1) {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', 'https://dog.ceo/api/breed/' + breed + '/images/random');
+    xhr.responseType = 'json';
+    xhr.addEventListener('load', function () {
+
+      var newPic = renderPic(xhr.response);
+      $pictureList.appendChild(newPic);
 
-  });
-  xhr.send();
+    });
+    xhr.send();
+  } else if (picViewNumber === 2) {
+    var xhrtwo = new XMLHttpRequest();
+    xhrtwo.open('GET', 'https://dog.ceo/api/breed/' + breed + '/images/random/2');
+    xhrtwo.responseType = 'json';
+    xhrtwo.addEventListener('load', function () {
+      for (var a = 0; a < xhrtwo.response.message.length; a++) {
+        var newPicTwo = renderMorePic(xhrtwo.response.message[a]);
+        $pictureList.appendChild(newPicTwo);
+      }
 
+    });
+    xhrtwo.send();
+  } else if (picViewNumber === 3) {
+    var xhrthree = new XMLHttpRequest();
+    xhrthree.open('GET', 'https://dog.ceo/api/breed/' + breed + '/images/random/3');
+    xhrthree.responseType = 'json';
+    xhrthree.addEventListener('load', function () {
+      for (var b = 0; b < xhrthree.response.message.length; b++) {
+        var newPicThree = renderThreePic(xhrthree.response.message[b]);
+        $pictureList.appendChild(newPicThree);
+      }
+
+    });
+    xhrthree.send();
+  }
```

Comment:
If you revise your code to use one function for rendering images, you can make the only work needed for the conditional here to change the URL to send the data to, and the rest of this should be the same for all cases, I believe